appengine-java11/micronaut-helloworld/pom.xml

```diff
@@ -22,11 +22,11 @@
   The parent pom defines common style checks and testing strategies for our samples.
   Removing or replacing it should not affect the execution of the samples in anyway.
   -->
-  <parent>
+  <!-- <parent>
```

Comment:
Can you add the relevant issue number for context?

Reply:
The Github issue? #1649

Reply:
Yes please - just so we know why it's disabled.